pddl/Pddl.g4

```diff
@@ -1,127 +1,208 @@
 /**
- * PDDL grammar for ANTLR v3
+ * PDDL grammar for ANTLR v4
+ *
  * Zeyn Saigol
  * School of Computer Science
  * University of Birmingham
  *
- * $Id: Pddl.g 120 2008-10-02 14:59:50Z zas $
+ * modified for v4 and made it language agnostic by
+ * Hernán M. Foffani
+ * Now part of the project
+ * https://bitbucket.org/hfoffani/pddl-parser
+ *
+ *
+ * Modified to add Multi-Agent PDDL (MAPDDL),
+ * following most of the specification by Kovacs, 2012.
+ * https://home.mit.bme.hu/~dkovacs/pubs/d.l.kovacs_2012_ICAPS-WIPC.pdf
+ * Agents have the capability (in this implementation) to carry out
+ * whichever actions they please, future work sees the addition of
+ * agents to the goal condition.
+ * Stewart Anderson 2024-03-29
+ * (stewart.anderson@abdn.ac.uk)
+ *
+ * This file carries the same license from the original.
  */
 
-// $antlr-format alignTrailingComments true, columnLimit 150, minEmptyLines 1, maxEmptyLinesToKeep 1, reflowComments false, useTab false
-// $antlr-format allowShortRulesOnASingleLine false, allowShortBlocksOnASingleLine true, alignSemicolons hanging, alignColons hanging
+ // $antlr-format alignTrailingComments true, columnLimit 150, minEmptyLines 1, maxEmptyLinesToKeep 1, reflowComments false, useTab false
+ // $antlr-format allowShortRulesOnASingleLine false, allowShortBlocksOnASingleLine true, alignSemicolons hanging, alignColons hanging
```

Comment:
I'm not sure why these formatting rules were changed. Please go to https://github.com/mike-lischke/antlr-format/blob/main/cli/ReadMe.md and install the antlr-formatter. Then, reformat this grammar `antlr-format *.g4`.

Reply:
The formatting rules weren't changed - they just moved down ~15 lines?

Apologies for not formatting, I've now formatted the file.